notebooks/Notebook1.md

```diff
@@ -18,17 +24,35 @@ exciting milestones, preliminary results, etc. Feel free to include images
+
 **What questions do you have for your critique partners? How can they best help
 you?**
 
+First, I want to know if my goal realignment makes sense for the language; is it really better that my language should be more fully-featured at the expense of its readability?
+Second, I have a design conundrum: I'm not sure whether to have a few classes with a lot of functionality, all of which will not necessarily be used in every program, or many classes with more specific functionality. I'm leaning towards the former, as it feels more intuitive: someone programming in the language is more likely to understand a Card class, rather than a CardWithTextAndFlavor class and a CardWithPowerAndToughness class. If they need more specific options, or need additional features, the language will still support subclassing and inheritance.
+
 **How much time did you spend on the project this week? If you're working in a
 team, how did you share the work?**
 
+Not totally sure, but somewhere between 7 and 9 hours, probably around 8 (not including the time I spent working on this notebook).
+
 **Compared to what you wrote in your contract about what you want to get out of this
 project, how did this week go?**
+
+I feel like I'm on track for where I wanted to be with the project, but the design phase has been far tougher than I expected it to be. To be honest, I hadn't really appreciated how difficult rules enforcement was going to be, which is something that I figured was going to be fairly easy. Honestly, I still haven't fully decided how deeply rules enforcement is going to be embedded in this version of the DSL; it's extremely finicky, and I'm worried its slowing down the project. However, for this to be viable for a computer to play, it needs to have rules enforcement: a computer playing chess that didn't understand the rules of chess, but knew how to win, would try to take the King on the first move every time; that makes the main goal of my project largely null and void.
+I'm not sure whether the project will be at all close to the point where I want it to be at the end of these four weeks; however, I do feel like I am putting in good work (and a lot of time), and learning some difficult lessons about language design. I think I'm still on-track for an A on the project, and I have the distinct feeling I'll be putting some work in on it over the summer- it's been a blast to work on so far, despite the difficulties I've been having. Hopefully, critique groups will clear up my uncertainty a bit, and I'm excited to jump back into Scala.
```

Comment:
mood. I think we will get farther then we think, but my other projects may steal my time